charts/synthetics-job-manager/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+name: synthetics-job-manager
+description: New Relic Synthetics Containerized Job Manager
+type: application
+version: 1.0.0
+appVersion: release-155
+home: https://github.com/orgs/newrelic/teams/proactive-monitoring
+maintainers:
+  - name: Philip-R-Beckwith
+  - name: sshah-nr
+  - name: vkasanneni-nr
+  - name: marcusperezNR
+  - name: mderemer-nr
+  - name: kaschaefer-nr
+  - name: sadafarshad
+  - name: lromer22
+    email: team_proactive-monitoring@newrelic.com
+keywords:
+  - synthetics
+  - synthetics-job-manager
+  - newrelic
+dependencies:
```

Comment:
has `helm dependency update ...` been run for this chart? I don't see the `chart.lock` file for it, but it might be because of nested charts approach.